python/hidet/lang/__init__.py

```diff
@@ -31,6 +31,23 @@
 from hidet.lang.constructs.type import tensor, tensor_pointer, as_tensor_pointer, register_tensor, shared_tensor
 
 
+from hidet.ir.primitives.cpu import (
+    avx_f32x4_broadcast,
+    avx_f32x4_fmadd,
+    avx_f32x4_load,
+    avx_f32x4_store,
+    avx_f32x4_setzero,
+)
+from hidet.ir.primitives.cpu import (
+    avx_f32x8_broadcast,
+    avx_f32x8_fmadd,
+    avx_f32x8_load,
+    avx_f32x8_store,
+    avx_f32x8_setzero,
+)
+from hidet.ir.primitives.cpu import avx_free, avx_malloc, x86_memcpy, x86_memset, aligned_alloc
+
+
```

Comment:
Let's create a python module called "cpu.py" under python/hidet/lang to put all cpu-related primitives.

Reply:
And remove the /python/hidet/lang/avx.py empty file.